lib/double_entry/reporting/line_aggregate.rb

```diff
@@ -2,8 +2,8 @@
 module DoubleEntry
   module Reporting
     class LineAggregate < ActiveRecord::Base
-      def self.aggregate(function, account, code, range, named_scopes)
-        collection_filter = LineAggregateFilter.new(account, code, range, named_scopes)
+      def self.aggregate(function, account, code, range, named_scopes, partner_account)
```

Comment:
Would be good to have keyword arguments here. This is a large number for positional parameters.